### PR TITLE
Clean up references to Glossary

### LINKED ISSUE #4804: Clean up references to Glossary
context: #3824

1. the glossary is still in place
2. a lot of redirects point to it
3. a lot of articles still point to the glossary

what needs to be done:
- [ ] redirects must point to the new articles;
- [x] direct links to the glossary must point to the new articles;
- [ ] the glossary needs to be an [index article](https://github.com/ppy/osu-wiki/blob/master/wiki/Article_styling_criteria/Formatting/en.md#index-articles)

_Originally posted by @TicClick in https://github.com/ppy/osu-wiki/issues/3824#issuecomment-762525385_

### PR DESCRIPTION
**Pull request by DoryanR**
part of #4804

Fixed links to sub-sections of the Glossary, to the `Glossary` article, and other links that are invalid.

The only thing left is to know what to do with the articles in `/wiki/Glossary`, if it is possible to put them in other folders, because I think they will have their place in other places.

### CODE REVIEW

File: wiki/Beatmap/Category/zh.md [hunk 1]
```diff
@@ -9,27 +9,29 @@ outdated_since: f4d914b0b826e19e50caeea90f58557065e9e0b7
...
+    - [Qualified](#qualified)
+    - [Ranked](#ranked)
+    - [Loved](#loved)
 
 ### Graveyard
 
 状态为 Graveyard 的谱面是被谱面创建者弃更了的谱面。只要一个在 [Work in Progress / Pending](#work-in-progress-和-pending) 类别中的谱面超过4周没有任何更新，它就会被自动地移至 Graveyard 类别中。类别为 Graveyard 的谱面没有排行榜，也不会影响用户成绩。但是，它们仍可以被正常的下载并游玩。这些谱面在选歌界面中使用问号（![](/wiki/shared/status/graveyard.png)）图标。
 
-当一个谱面变为Graveyard后，它仍可以被“复活” (如 重新变回[Pending](#work-in-progress-和-pending) ）。这可以由使用 [谱面提交系统](/wiki/Glossary#bss) 更新谱面达成。
+当一个谱面变为Graveyard后，它仍可以被“复活” (如 重新变回[Pending](#work-in-progress-和-pending) ）。这可以由使用 [谱面提交系统](/wiki/Beatmap_ranking_procedure) 更新谱面达成。
```
Comment (TicClick) [inline wiki/Beatmap/Category/zh.md:30]: this should be a link to `/wiki/Submission`

File: wiki/Beatmap_Editor/Compose/en.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ![Compose for osu!mania using osu!screenshot (only the external upper menu is not shown)](img/ComposeMania.jpg "Compose for osu!mania using osu!screenshot (only the external upper menu is not shown)")
 
-**Compose** is the place where a [mapper](/wiki/Glossary) spent most of the time mapping out his/her design with respect to timeline after setting the BPM and Timing Sections. This is also the place where players can practice a ranked difficulty and inspect its design akin to DDR's Training mode.
+**Compose** is the place where a [mapper](/wiki/Modding/Modder) spent most of the time mapping out his/her design with respect to timeline after setting the BPM and Timing Sections. This is also the place where players can practice a ranked difficulty and inspect its design akin to DDR's Training mode.
```
Comment (TicClick) [inline wiki/Beatmap_Editor/Compose/en.md:7]: as we don't have an article about the mapper term (and it's unlikely that we need to), this should link to `/wiki/Beatmapping`, same for the translations of "Compose"

File: wiki/Beatmap_Editor/es.md [hunk 3]
```diff
@@ -9,6 +9,6 @@ outdated: true
 El **beatmap editor** (editor de beatmap) es una interfaz gráfica en osu! que sirve para crear [beatmaps](/wiki/Beatmap). Está separado en 4 modos:
 
 - [Compose](Compose) (Componer): Aquí es donde los [hit objects](/wiki/Hit_Objects) son puestos y editados. La mayoría del tiempo empleado en el editor pasa aquí.
-- [Design](Design) (Diseño): Aquí es donde el [storyboard](/wiki/Storyboards) es hecho. La mayoría de los [mappers](/wiki/Glossary) solo usan el botón "Background Image/Video" aquí.
-- [Timing](Timing) (Tempo): Aquí es donde el [BPM](Timing) y el [offset](/wiki/Glossary) de la canción son ingresados. Esta ventana también controla los [hitsounds](/wiki/Glossary) si estos no son constantes a lo largo del beatmap.
+- [Design](Design) (Diseño): Aquí es donde el [storyboard](/wiki/Storyboards) es hecho. La mayoría de los [mappers](/wiki/Modding/Modder) solo usan el botón "Background Image/Video" aquí.
```
Comment (TicClick) [inline wiki/Beatmap_Editor/es.md:12]: again, this should reference `/wiki/Beatmapping`

**Review by TicClick — CHANGES_REQUESTED** [review]
looks ok otherwise -- is there anything else that links to Glossary besides the articles you edited?

**Comment by TicClick** [comment]
this also only resolves a part of #4804 -- I'd rather have several medium-sized pull requests to review (one for each checkbox item)

**Comment by DoryanR** [comment]
> is there anything else that links to Glossary besides the articles you edited?

I have quite a few more articles to do, about 220 with links pointing to the Glossary.

> this also only resolves a part of #4804 -- I'd rather have several medium-sized pull requests to review (one for each checkbox item)

Ok, no problem

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Modding/fr.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Que vous soyez un mappeur débutant ou un peu plus expérimenté, le modding est
...
+Tout d'abord, téléchargez une map dans la partie [Pending Beatmap](/wiki/Beatmap) ou [Work In Progress/Help](/wiki/Beatmap/Category#work-in-progress-and-pending) du forum (et non les parties [Ranked](/wiki/Beatmap)/[Approved](/wiki/Beatmap) !). Vous pouvez choisir des maps au hasard ou ouvrir une modding queue dans la sous-partie **Modding Queue** en dessous de [Work In Progress/Help](/wiki/Beatmap/Category) et accepter des demandes de mods. Si vous avez l'intention d'ouvrir une modding queue, n'hésitez pas à jeter un coup d'oeil à certains d'entre eux pour vous donner une idée générale de leur fonctionnement.
```
Comment (TicClick) [inline wiki/Modding/fr.md:21]:
```suggestion
Tout d'abord, téléchargez une map dans la partie [Pending Beatmap ou Work In Progress/Help](/wiki/Beatmap/Category#work-in-progress-and-pending) du forum (et non les parties [Ranked](/wiki/Beatmap/Category#ranked)/[Approved](/wiki/Beatmap/Category#approved) !). Vous pouvez choisir des maps au hasard ou ouvrir une modding queue dans la sous-partie **Modding Queue** en dessous de [Work In Progress/Help](/wiki/Beatmap/Category#work-in-progress-and-pending) et accepter des demandes de mods. Si vous avez l'intention d'ouvrir une modding queue, n'hésitez pas à jeter un coup d'oeil à certains d'entre eux pour vous donner une idée générale de leur fonctionnement.
```

File: wiki/Modding/zh.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ outdated: true
 
 ## 基本修图方法
 
-首先，从论坛上下载一个谱面，[Pending Beatmaps 里的或者](/wiki/Beatmap) [Work In Progress/Help 的都可以](/wiki/Beatmap/Category)(唯独不要下载 [Ranked/Approved Beatmaps](/wiki/Beatmap) 里的！因为那些是已经审核通过上架了的谱面)。你可以从之前两个页面中随机选一个谱面来看，或者在 [Work In Progress/Help 里的](/wiki/Beatmap/Category) **Modding Queue** 页面开一个自己的 Modding queue（修图队列）然后接受他人的修图请求。如果你想开一个修图队列，读一读那个页面里的其他队列你就知道该怎么做了。
+首先，从论坛上下载一个谱面，[Pending Beatmaps 里的或者](/wiki/Beatmap) [Work In Progress/Help 的都可以](/wiki/Beatmap/Category)(唯独不要下载 [Ranked/Approved Beatmaps](/wiki/Beatmap) 里的！因为那些是已经审核通过上架了的谱面)。你可以从之前两个页面中随机选一个谱面来看，或者在 [Work In Progress/Help 里的](/wiki/Beatmap/Category#work-in-progress-and-pending) **Modding Queue** 页面开一个自己的 Modding queue（修图队列）然后接受他人的修图请求。如果你想开一个修图队列，读一读那个页面里的其他队列你就知道该怎么做了。
```
Comment (TicClick) [inline wiki/Modding/zh.md:17]:
```suggestion
首先，从论坛上下载一个谱面，[Pending Beatmaps 里的或者](/wiki/Beatmap/Category#work-in-progress-and-pending) [Work In Progress/Help 的都可以](/wiki/Beatmap/Category#work-in-progress-and-pending)(唯独不要下载 [Ranked/Approved Beatmaps](/wiki/Beatmap/Category#ranked) 里的！因为那些是已经审核通过上架了的谱面)。你可以从之前两个页面中随机选一个谱面来看，或者在 [Work In Progress/Help 里的](/wiki/Beatmap/Category#work-in-progress-and-pending) **Modding Queue** 页面开一个自己的 Modding queue（修图队列）然后接受他人的修图请求。如果你想开一个修图队列，读一读那个页面里的其他队列你就知道该怎么做了。
```

**Review by TicClick — CHANGES_REQUESTED** [review]
a few last bits and that'll be it

**Review by TicClick — APPROVED** [review]

**Pull request merged**